.github/workflows/ci.yml

```diff
@@ -0,0 +1,32 @@
+name: Ruby
+
+on:
+  [push, pull_request]
+
+permissions:
+  contents: read
+
+jobs:
+  test:
+
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        ruby-version: ['2.6', '2.7'] #repo doesn't yet support 3.0 :upside_down_face:
+
+    steps:
+    - uses: actions/checkout@v3
+    - name: Set up Ruby
+      uses: ruby/setup-ruby@v1
+      with:
+        ruby-version: ${{ matrix.ruby-version }}
+        bundler-cache: true # runs 'bundle install' and caches installed gems automatically
+    - name: Setup Test Env
+      run: |
+        touch .env
+        echo BT_ENVIRONMENT='${{ secrets.BT_ENVIRONMENT }}' >> .env
+        echo BT_MERCHANT_ID='${{ secrets.BT_MERCHANT_ID }}' >> .env
+        echo BT_PUBLIC_KEY='${{ secrets.BT_PUBLIC_KEY }}' >> .env
+        echo BT_PRIVATE_KEY='${{ secrets.BT_PRIVATE_KEY }}' >> .env
```

Comment:
When Sarah set these up for Android a while back she mapped env vars to secrets using [this method](https://github.com/braintree/braintree_android/blob/11d8d282bfe6744fb5f0c0a5e9120b8cd223f8cf/.github/workflows/release.yml#L27). The run step output could get logged to the console.

Not sure how this would affect the `bundle exec rake spec` step, but it could add an additional layer of security.

Reply:
That's a good callout! Let me try updating with this feedback and see how that looks

Reply:
Ok so we ran into a little bump because the example repos are set up for easy configuring with a .env file, trying load env variables using env vs the current way in the `run` steps results in issues loading credentials because the app is looking for a .env file.

Given that this is a demo app, and the secrets stored are not linked to anything production, I'm ok merging as is.